ONSdigital/dp-frontend-filter-dataset-controller
# Update controller to match ux

### What

- Add stubbed data for cmd breadcrumbs and metadata footer as well as to disable search bar on these pages

### How to review

- Start filter-dataset-controller and renderer service (on branch feature/add-cmd-ux)
- navigate from http://localhost:20001/datasets/1234/editions/5678/versions/2017 through journey
- You should notice search bar is hidden and metadata footer and breadcrumbs on middle page exist

### Who can review

Anyone
